internal/operands/metrics/resources.go

```diff
@@ -2,18 +2,31 @@ package metrics
 
 import (
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	v1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	rbac "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
 const (
-	PrometheusRuleName       = "prometheus-k8s-rules-cnv"
-	runbookURLBasePath       = "http://kubevirt.io/monitoring/runbooks/"
-	severityAlertLabelKey    = "severity"
-	partOfAlertLabelKey      = "kubernetes_operator_part_of"
-	partOfAlertLabelValue    = "kubevirt"
-	componentAlertLabelKey   = "kubernetes_operator_component"
-	componentAlertLabelValue = "ssp-operator"
+	PrometheusRuleName           = "prometheus-k8s-rules-cnv"
+	MonitorNamespace             = "openshift-monitoring"
```

Comment:
Will it only work in OpenShift?
I think it should work for both `monitoring` and `openshift-monitoring` namespaces...

Reply:
this will only work in openshift (the operator is working with templates that only work on openshift anyway)

Reply:
and it should run on okd, since it has the same namespace (https://docs.okd.io/latest/monitoring/monitoring-overview.html#default-monitoring-components_monitoring-overview)